.evergreen/config.yml

```
@@ -1824,87 +1830,94 @@ axes:
   - id: os-ssl-legacy
     display_name: OS
     values:
-      - id: "ubuntu1404-go-1-15"
+      - id: "ubuntu1404-go-1-16"
         display_name: "Ubuntu 14.04"
         run_on: ubuntu1404-test
         variables:
-          GO_DIST: "/opt/golang/go1.15"
+          GO_DIST: "/opt/golang/go1.16"
+          # The GCC toolchain that comes with Ubuntu 14.04 is incompatible with the standard binary
+          # release of Go 1.16. As a result, Cgo builds on Ubuntu 14.04 fail with a linking error
+          # (see https://github.com/golang/go/issues/43996). On Ubuntu 14.04, we only run tests for
+          # server v2.6, which doesn't support client-side encryption. Exclude all build tags to
```

Ubuntu 14.04 also runs against the server v3.0.
```suggestion
          # server v2.6 and v3.0, which doesn't support client-side encryption. Exclude all build tags to
```

In case it is helpful, the support of server versions on different platforms found on: https://docs.mongodb.com/manual/administration/production-notes/
The version selector menu in the top left can be used to look at older versions of those notes. From the [4.2 version of those notes](https://docs.mongodb.com/v4.2/administration/production-notes/#x86-64), support of Ubuntu 14.04 was dropped in 4.2.

---

Thanks for the links! I've updated the comment to reflect that we test v2.6 and 3.X on Ubuntu 14.04.